DistTasks.yml

```diff
@@ -184,7 +184,7 @@ tasks:
       #
       # Until there is a fix released we must use a recent gcc for Linux_ARMv6 build, so for this
       # build we select the debian10 based container.
-      CONTAINER_TAG: "{{.GO_VERSION}}-armel-debian10"
+      CONTAINER_TAG: "{{.GO_VERSION}}-armel-debian9"
```

Comment:
why this was necessary?

Reply:
I copied it from the [`DistTask.yml`](https://github.com/arduino/arduino-cli/blob/master/DistTasks.yml) in the Arduino CLI repo.

I guess that @MatteoPologruto tried to go back to debian9 to see if the problem in the comment was fixed (and it seems fixed indeed) but didn't update the comment.
We should:
* Update the comment (remove it)
* Update the template in the asset repo